issuegenerator/main.go

```diff
@@ -115,9 +179,9 @@ type reportGenerator struct {
 func (rg *reportGenerator) getRequiredEnv() {
```

Comment:
i think the doc string for this method needs an update

Reply:
Great catch, this also made me research the default env variables from Github actions and realize we can re-use some of them :)

I'll add a commit updating the variables and doc string